lxd/images.go

```diff
@@ -1153,6 +1155,61 @@ func autoUpdateImagesInProject(ctx context.Context, d *Daemon, project string) e
 		return errors.Wrap(err, "Unable to retrieve the list of images")
 	}
 
+	var nodes []api.ClusterMember
+
+	// Wait up to 30 seconds for all nodes to get online.
```

Comment:
Why is all this logic needed?

I would have expected us to spread the image update over all online nodes by simply fetching the current nodes list, removing the offline node and then picking a "random" index in there for each image. If a node comes back online, then things will get reshuffled, but that should be fine.

Reply:
The images are also refreshed on startup. I noticed that all cluster nodes are marked as offline when this code in run and take some time to appear to be online. That's the reason I added this logic.